packages/celotool/src/cmds/geth/start.ts

```diff
@@ -0,0 +1,208 @@
+import yargs from 'yargs'
+
+import { addCeloGethMiddleware } from 'src/lib/utils'
+import {
+  AccountType,
+  getPrivateKeysFor,
+  getValidatorsInformation,
+  privateKeyToPublicKey,
+} from '../../lib/generate_utils'
+import { getEnodeAddress, runGethNodes } from '../../lib/geth'
+import { GethInstanceConfig } from '../../lib/interfaces/geth-instance-config'
+import { GethRunConfig } from '../../lib/interfaces/geth-run-config'
+import { GethArgv } from '../geth'
+
+export const command = 'start'
+export const describe = 'command for running geth'
+
+interface StartArgv extends GethArgv {
+  networkId: string
+  syncMode: string
+  mining: boolean
+  blockTime: number
+  nodekeyhex: string
+  minerGasPrice: number
+  port: number
+  rpcport: number
+  wsport: number
+  verbosity: number
+  verbose: boolean
+  instances: number
+  purge: boolean
+  withProxy: boolean
+  ethstats: string
+}
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloGethMiddleware(argv)
+    .option('network-id', {
+      type: 'string',
+      description: 'network id',
+      default: '1101',
+    })
+    .option('sync-mode', {
+      choices: ['full', 'fast', 'light', 'ultralight'],
```

Comment:
With Geth 1.9 `lightest` will be the new name for `ultralight` as `ultralight` is already a Geth 1.9 sync mode. Might as well add it here now.
```suggestion
      choices: ['full', 'fast', 'light', 'ultralight', 'lightest'],
```